_posts/2018-03-28-building-coreboot-on-minnowboard.md

```diff
@@ -0,0 +1,244 @@
+Building coreboot on MinnowBoard
```

Comment:
@mscibisz we should have some notes how correct blog post should look like. Please prepare such documentation for internal use.

@BartekLew we have to add header and this post should be in post.md file in main directory otherwise it will not be automatically fetched by our wordpress plugins.

Reply:
is there some header format specification? Where is this `post.md` I can't find it.

Reply:
@BartekLew please take a look at this PR: https://github.com/3mdeb/news-and-ideas/pull/18/files

Reply:
:+1: